src/unittests/f32_interval.spec.ts

```diff
@@ -28,6 +29,16 @@ function arrayToInterval(bounds: [number, number]): F32Interval {
   return new F32Interval(begin, end);
 }
 
+/** @returns a number one ULP greater than the provided number */
+function plusOneULP(n: number): number {
+  return n + Math.max(oneULP(n, false), oneULP(n, true));
```

Comment:
This could be simplified with https://github.com/gpuweb/cts/pull/1518, right?

Reply:
That is correct, rebased and simplified